drivers/nutdrv_qx.c

```diff
@@ -790,7 +841,7 @@ static int	krauler_command(const char *cmd, char *buf, size_t buflen)
 }
 
 /* Fabula communication subdriver */
-static int	fabula_command(const char *cmd, char *buf, size_t buflen)
+static int	_fabula_command(const char *cmd, char *buf, size_t buflen, char hunnox_patch)
```

Comment:
I think it is not worth adding a `hunnox_patch` parameter which is only for Hunox 850. A function should be use for other protocols. Do you have any other options?

Reply:
I thought that duplicating the entire function was a bad idea. But maybe it's the safest way to avoid problems for other UPSs? What do you recommend?

Reply:
My stance on this wavered over time, but in the end I guess making an explicit "hunnox_command" that happens to repeat some of the other code can be more readable and maintainable, and would clearly "protect" support of different devices from these and later changes in not-their subdriver.

It may be or not be possible and practical that shared code can be de-duplicated into new routines called from various places, as a separate effort.